src/components/Navigator/NavigatorCard.vue

```diff
@@ -1121,6 +1121,10 @@ $navigator-head-background-active: var(--color-fill-tertiary) !default;
   @include breakpoint(medium, nav) {
     padding-bottom: $nav-menu-items-ios-bottom-spacing;
   }
+  // hack to allow scrolling on Safari
```

Comment:
Can you explain the "hack" better in this comment? I think this will be very confusing to anyone else reading this in the future since it's not clear why we're doing the translation without a different position. Your PR mentions that this affects the stacking context for Safari—it would be good to describe the specific problem in this comment and how this actually resolves that. Thanks!

Reply:
Here is where I got the fix from - https://github.com/Akryum/vue-virtual-scroller/issues/387#issuecomment-833304471